src/model-viewer-base.ts

```diff
@@ -325,6 +326,31 @@ export default class ModelViewerElementBase extends UpdatingElement {
     return this[$canvas].toDataURL(type, encoderOptions);
   }
 
+  async toBlob(mimeType?: string, qualityArgument?: number): Promise<Blob> {
+    return new Promise(async (resolve, reject) => {
+      if ((this[$canvas] as any).msToBlob) {
+        // NOTE: msToBlob only returns image/png
+        // so ensure mimeType is not specified (defaults to image/png)
+        // or is image/png, otherwise fallback to using toDataURL on IE.
+        if (!mimeType || mimeType === 'image/png') {
```

Comment:
Let me know if we want to avoid nested if statements and whether the note is necessary/useful.

Reply:
Also should `'image/png'` be a constant?

Reply:
Exceeds expectations! 🙌